keyboards/aeboards/ext65rev2/boards/ST_STM32F072B_DISCOVERY/board.c

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Board files should be removed and refactored to use <https://docs.qmk.fm/#/platformdev_chibios_earlyinit>